### DISCUSSION TITLE
Claude Desktop: Subscription vs. API model

### DISCUSSION THREAD

**Discussion by rekdt** [modelcontextprotocol/.github#62] [Q&A]
### Pre-submission Checklist

- [X] I have checked that this question would not be more appropriate as an issue in a specific repository
- [X] I have searched existing discussions and documentation for answers

### Question Category

- [ ] Protocol Specification
- [ ] SDK Usage
- [ ] Server Implementation
- [ ] General Implementation
- [ ] Documentation
- [ ] Other

### Your Question

Does anyone know if Claude Desktop in developer mode uses the subscription payment model or the API? I don’t see any logs in the API when the desktop uses function calling, so I’m assuming it’s limited to the subscription model? If that’s the case, is there a way to change it? It would be frustrating to be restricted to just the subscription model.

**Comment by jspahrsummers** [comment]
Sorry, this is not an Anthropic support channel. Please check https://support.anthropic.com.
